metadata/experiment.go

```diff
@@ -50,6 +50,8 @@ type ExperimentDefinition struct {
 	ClusterDefinition ClusterDefinition
 
 	ScenarioDefinition ScenarioDefinition
```

Comment:
Ah I see why you got confused before. We should remove these top-level cluster/scenario definitions.

Reply:
Sounds good